addons/account/views/account_payment_view.xml

```diff
@@ -313,7 +313,7 @@
                             <group>
                                 <field name="qr_code" invisible="1"/>
                                 <div attrs="{'invisible': [('qr_code', '=', False)]}" colspan="2" class="text-center">
-                                    <field name="qr_code" widget="html"/>
+                                    <field name="qr_code"/>
```

Comment:
> Bleh that'll teach me to check master then make the change in stable without checking again.
> 
> Although is there any reason for it not to be an html field, essentially backporting #122306?

Not that i know. @oco-odoo  ?

Reply:
Hmmmm ... Not that I remember, and indeed, it seems quite absurd. I don't see the added-value of doing that in stable, though. It's pure cleaning and nothing is broken, right ? This kind of change should typically go to master, especially since this one has been there for years, and fixing it in stable would mean fixing it older versions than 16.0.